src/components/CellsContainer/helpers.js

```diff
@@ -0,0 +1,92 @@
+import { create } from 'd3';
+
+export function getCellSize(totalCellsCount) {
+  switch (true) {
```

Comment:
Not sure that this is better, and more readeable, but... 
```javascript
cond([
    [(v) => v < 100, () => 38],
    [(v) => v < 500, () => 20],
    [(v) => v < 1000, () => 18],
    [(v) => v < 2000, () => 16],
    [() => true, () => 11]
  ])(700)
```
So you can just return it like 
```javascript
export const getCellSize = cond([
    [(v) => v < 100, () => 38],
    [(v) => v < 500, () => 20],
    [(v) => v < 1000, () => 18],
    [(v) => v < 2000, () => 16],
    [() => true, () => 11]
  ]);
```
Pure, without any variables inside! 😏

Reply:
I'm starting to like lodash:)

Reply:
it has no  switch (true) so I like it more! And it's pretty clean if I know cond.